rust/worker/src/blockstore/arrow/blockfile.rs

```diff
@@ -590,8 +597,14 @@ mod tests {
         Runtime::new().unwrap().block_on(async {
             let tmp_dir = tempfile::tempdir().unwrap();
             let storage = Storage::Local(LocalStorage::new(tmp_dir.path().to_str().unwrap()));
-            let blockfile_provider =
-                ArrowBlockfileProvider::new(storage, TEST_MAX_BLOCK_SIZE_BYTES);
+            let block_cache = Cache::new(&CacheConfig::Unbounded(UnboundedCacheConfig {}));
+            let sparse_index_cache = Cache::new(&CacheConfig::Unbounded(UnboundedCacheConfig {}));
```

Comment:
this is premature - but in the future would it makes sense for these caches to be aware of each other in any way? I.e. - we don't evict the sparse index if the blocks are touched etc. I think not, since they are always accessed together. Just had a thought so dumping it.